webpack.config.js

```diff
@@ -17,5 +18,15 @@ module.exports = {
       }
     ]
   },
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        { 
+		  from: 'node_modules/bpmn-js-token-simulation/assets', 

```

Comment:
I have some unpushed changes after checking out out locally

![image](https://user-images.githubusercontent.com/9433996/99072018-7f4c1b80-25b3-11eb-8448-0f2fe9adad6c.png)

Since this plugin is copying some stuff around we should ensure to
a) ignore the changes depending on the downstream token-simulation library
b) pushing them alongside the `client/client.bundle.js.map` changes

Reply:
I actually don't know why there are folders like css and client in the first place. This is basically binary data in a repository or am I missing something?

Reply:
We include the bundles to make the plugin usable right after download so the user does not have to rebuild the plugin on its own.

Reply:
Since we're not publishing the plugins via npm or something where we could easily push the bundles to, we just include them in the repository.

Reply:
And that's why there are uncommited changes in your workspace. I think it's wrong, but that's probably another issue or pull request. 

Alternatively, we might restore webpack.config.js and just commit its result. However, there will then be more errors like #18 in the future.

Reply:
Could not reproduce the thing with uncommited changes.

The solution itself is solid. Let's monitor and fix what's left in the future.